sql/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -305,15 +310,18 @@ public void testStatsTableFailure() throws Exception {
             conn.createStatement().execute("set global stats.enabled=true");
             conn.prepareStatement("create table t (a integer not null, b string)").execute();
 
+            String uniqueId = UUID.randomUUID().toString();
+            String stmtStr = "insert into t(a,b) values(null, '" + uniqueId + "')";
+            String stmtStrWhere = "insert into t(a,b) values(null, ''" + uniqueId + "'')";
             try {
-                conn.prepareStatement("insert into t(a,b) values(null, 'test')").execute();
-            } catch (Exception e) {}
-
-            ResultSet resultSet = conn.prepareStatement("select stmt, error from sys.jobs_log order by ended desc;")
-                .executeQuery();
-            assertThat(resultSet.next(), is(true));
-            assertThat(resultSet.getString(1), is("insert into t(a,b) values(null, 'test')"));
-            assertThat(resultSet.getString(2), is("Cannot insert null value for column a"));
+                conn.prepareStatement(stmtStr).execute();
+            } catch (Exception e) {
+                ResultSet resultSet = conn.prepareStatement("select stmt, error from sys.jobs_log " +
```

Comment:
maybe you could add a `fail` after line 317 just to assure that the test won't pass if the error doesn't happen